packages/components/src/badge/stories/index.story.tsx

```diff
@@ -8,12 +8,12 @@ import type { Meta, StoryObj } from '@storybook/react';
  */
 import Badge from '..';
 
-const meta = {
+const meta: Meta< typeof Badge > = {
 	component: Badge,
 	title: 'Components/Containers/Badge',
 	id: 'components-badge',
 	tags: [ 'status-private' ],
-} satisfies Meta< typeof Badge >;
+};
```

Comment:
Normalizing this declaration to our standard format (not TS `satisfies`) for easier AST parsing.

Reply:
Thanks for this 👍 The `satisfies` syntax seemed to cause some issues when parsing files for component usage as well.